instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/patches/launcher.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Sidekiq
+      module Patches
+        # The Launcher module contains the instrumentation for the Sidekiq heartbeat
+        module Launcher
+          def ❤ # rubocop:disable Naming/MethodName
+            if config[:trace_launcher_heartbeat]
```

Comment:
- Do we want to establish some sort of naming convention for config options?
- I'm in the opinion that this should be off by default to provide a better use experience, this will generate a non trivial amount of spans.

Reply:
> * Do we want to establish some sort of naming convention for config options?

We don't have many config options yet, so you're in early & get to establish the conventions 😄 

> * I'm in the opinion that this should be off by default to provide a better use experience, this will generate a non trivial amount of spans.

👍